### PR TITLE
[Enhancement] Add ability to docs updater to push to docs.meshery.io

### LINKED ISSUE #8381: Meshery Updater should push to docs.meshery.io
#### Current Behavior
Inside `/scripts`, the component updater is used to update components' metadata, such as SVGs, icons, and other details, for meshery.io and layer5.io

#### Desired Behavior
It should also have the ability to push the same to docs.meshery.io

#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Boombag0607**
**Notes for Reviewers**

This PR fixes #8381 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: scripts/component_updater/main.go [hunk 1]
```diff
@@ -212,6 +213,7 @@ func docsUpdater(output []map[string]string) {
 		modelName := strings.TrimSpace(out["model"])
 		pathToIntegrationsLayer5, _ := filepath.Abs(filepath.Join("../../../", pathToIntegrationsLayer5, modelName))
 		pathToIntegrationsMeshery, _ := filepath.Abs(filepath.Join("../../../", pathToIntegrationsMeshery))
+		pathToIntegrationsMesheryDocs, _ := filepath.Abs(filepath.Join("../../", pathToIntegrationsMesheryDocs, "assets/img", modelName))
```
Comment (leecalcote) [inline scripts/component_updater/main.go:216]:
```suggestion
		pathToIntegrationsMesheryDocs, _ := filepath.Abs(filepath.Join("../../", pathToIntegrationsMesheryDocs, "assets/img/meshmodel/", modelName))	
```

**Comment by leecalcote** [comment]
@Boombag0607 I updated the file path to where these icons should be placed.

**Comment by leecalcote** [comment]
Is this ready for merge? @Boombag0607 @MUzairS15

**Comment by Boombag0607** [comment]
> Is this ready for merge? @Boombag0607 @MUzairS15

Yes.

**Pull request merged**